### DISCUSSION TITLE
License validity question

### DISCUSSION THREAD

**Discussion by bladepoint6969** [Riverside-Software/sonar-openedge#1133] [Q&A] [answered]
Is the SonarQube Server ID the major factor when checking license validity for the CABL rules, or is the SonarQube version also a factor?

For example, if our deployment was upgraded from 9.9 LTA to 10.6, would our rules license still be valid, or will a new one need to be generated?

**Comment by gquerret** [comment, marked as answer]
The CABL license is tied to the server ID only. The server ID depends on the DB system: https://docs.sonarsource.com/sonarqube/latest/instance-administration/license-administration/

The SQ instance can be upgraded to 10.5 without changing the CABL license. Note that SQ 10.6 was released 3 days ago, and is not yet verified with CABL.
